.github/workflows/e2e-tf-deployment.yaml

```diff
@@ -40,9 +40,11 @@ jobs:
 
     steps:
       - name: Checkout Helm charts
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.head_ref }}
```

Comment:
Did we really miss this? At some point, I believe we have verified that we are checking out the branch and testing the changes. Maybe it got lost in the ether.

Good catch.

Reply:
It was ok for branches in the repo, but not forks.